en/css/README.md

```diff
@@ -45,9 +48,13 @@ First, we need to create a directory to store our static files in. Go ahead and
 
 Open up the `mysite/settings.py` file, scroll to the bottom of it and add the following lines:
 
-    STATICFILES_DIRS = (
-        os.path.join(BASE_DIR, "static"),
-    )
+<!--TODO: this will conflict with the STATIC_ROOT specified earlier -->
```

Comment:
oops, sorry.  will review the PR line-by-line and make sure it's just syntax-highlights.